src/lib/scheduler/node_queue_scheduler.hpp

```diff
@@ -12,63 +12,53 @@ namespace hyrise {
 /*
  * GENERAL SCHEDULING CONCEPT
  *
- * Everything that needs to be processed is encapsulated in tasks. For example, in the context of the database
- * the OperatorTasks encapsulates database operators (here, it only encapsulates the execute function).
- * A task will then be pushed by a Scheduler into a TaskQueue and pulled out by a Worker to be processed.
+ * Everything that needs to be processed is encapsulated in tasks. For example, in the context of the database, the
+ * OperatorTasks encapsulates database operators (here, it only encapsulates the execute function). A task will be
```

Comment:
`, the/an OperatorTask encapsulates a database operator` OR `, OperatorTasks encapsulate database operators`

Reply:
(singular/plural also for text in parantheses)